.travis.yml

```diff
@@ -234,12 +234,12 @@ jobs:
       script:
        - travis_wait 30 mvn -B install -ntp -pl ":content-repository-elasticsearch-test" -am -Pall-tas-tests,elastic -Denvironment=default -DrunBugs=false "-Dsearch.engine.type=elasticsearch" "-Ddatabase.type=postgresql"
 
-#    - name: "Elasticsearch TAS tests | MySQL (Search API)"
-#      if: commit_message !~ /\[skip search\]/
-#      vm:
-#        size: large
-#      script:
-#        - travis_wait 30 mvn -B install -ntp -pl ":content-repository-elasticsearch-test" -am -Pall-tas-tests,elastic -Denvironment=default -DrunBugs=false "-Dsearch.engine.type=elasticsearch" "-Ddatabase.type=mysql"
+    - name: "Elasticsearch TAS tests | MySQL (Search API)"
+      if: commit_message !~ /\[skip search\]/
+      vm:
+        size: large
+      script:
+        - travis_wait 30 mvn -B install -ntp -pl ":content-repository-elasticsearch-test" -am -Pall-tas-tests,elastic -Denvironment=default -DrunBugs=false "-Dsearch.engine.type=elasticsearch" "-Ddatabase.type=mysql"
```

Comment:
"-Ddatabase.type=mysql" 

Is the variable name correct?
I don't know if that was supposed to be the one forcing DB implementation type for elasticsearch-connector, but that might be different 
https://github.com/Alfresco/alfresco-elasticsearch-connector/pull/430/files
alfresco-elasticsearch-reindexing/src/main/resources/application.properties

Reply:
If that's not it, then just resolve the comment